proxy/http2/Http2ConnectionState.cc

```diff
@@ -1449,16 +1449,21 @@ Http2ConnectionState::send_a_data_frame(Http2Stream *stream, size_t &payload_len
   return Http2SendDataFrameResult::NO_ERROR;
 }
 
-void
+/**
+ *  Returns true if the argument stream was deleted during
+ *  sending data frames
+ */
+bool
 Http2ConnectionState::send_data_frames(Http2Stream *stream)
 {
+  bool stream_deleted = false;
```

Comment:
minor nit-pick, but seems you could move this down in scope next to the "size_t len" declaration?